demos/appyx-navigation/web/src/jsMain/kotlin/com/bumble/appyx/navigation/Main.kt

```diff
@@ -123,6 +125,27 @@ private fun BlackContainer(content: @Composable () -> Unit) {
     }
 }
 
+@Composable
+private fun ProvideScopeConfiguration(
+    navigator: Navigator,
+    screenSize: ScreenSize,
+    body: @Composable () -> Unit
+) {
+    CompositionLocalProvider(
+        LocalDensity provides screenSize.calculateDensityFromScreenSize(),
+        LocalNavigator provides navigator,
+    ) {
```

Comment:
I'd actually prefer just merging this back to the call-site probably.

Reply:
Done